test/util.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+# This utility file contains functions that wrap commands to be tested and allow us to interface 
+# our command-line tests with JUnit on Jenkins. All wrapper functions run commands in a sub-shell 
+# and redirect all output. Tests in test-cmd *must* use these functions for testing.
+
+# HIGH-LEVEL WRAPPER FUNCTIONS: ASSUMING EXIT CODES
+
+# expect_success runs the cmd and expects an exit code of 0
+function os::test::expect_success() {
```

Comment:
Having monolithic, thousand-line `util` scripts smells really bad. This file includes *no* other imports and stands alone. Why bother with putting it into `hack/util.sh`?